src/eject/requiredDependencies.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * A list of dependencies which are installed into enclave, but will be necessary
+ * for a project if enclave is ejected.
+ * @type {string[]}
+ */
+var requiredDependencies = [
```

Comment:
That was my initial thought, but we have to differentiate between deps that need to be moved to the users project, and those that don't like `shelljs` and `chalk`.

We should actually probably do that and just move those other kinds of deps to devDependencies. duh